src/api/app/views/webui/project/_buildstatus.html.erb

```diff
@@ -21,13 +21,23 @@
                       </td>
                       <% index += 1 %>
                   <% end %>
-                  <td class="nowrap">
-                    <%= repo_status_icon(@repostatushash[repository][arch], @repostatusdetailshash[repository][arch])%> <%= arch %>
+                  <td class="arch" title="<%= "#{repository} summary" %>" >
+                    <div class="nowrap" style="margin: 0 0.5ex">
```

Comment:
this is actually the same as in [package build results](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/views/webui/package/_buildstatus.html.erb#L21). Should I change both to use same class & style?